src/features/NoResultsPage/styled.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const NoResultsText = styled.p`
+  font-weight: 600;
+  font-size: 33px;
+  letter-spacing: -0.5px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    font-size: 30px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    font-size: 22px;
+  }
+`;
+
+export const ImageContainer = styled.div`
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const Image = styled.img`
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    width: 200px;
+  }
+`;
```

Comment:
I think it's too small when width = 200px. Maybe we should change it to 440px and add media for small screens with width = 250px? What do you think? 🤔

Reply:
But I don't change the default size, only added media to medium screens so I'm not sure what you mean 😅 
I can change the size for small screens to 250px tho, looks good 👍